integration-tests/testsuite/src/test/java/io/apicurio/tests/multitenancy/MultitenantAuthIT.java

```diff
@@ -63,151 +32,9 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(MultitenantAuthIT.class);
 
-    private String adminRole = "sr-admin";
-    private String[] tenantRoles = new String[] {"sr-admin", "sr-developer", "sr-readonly"};
-
-    private RegistryFacade registryFacade = RegistryFacade.getInstance();
-
     @Test
+    @Disabled
```

Comment:
we should adapt this test instead of removing it, I can do it if you want

Reply:
I just disabled it until I adapt the other security stuff to the new situation. So what's missing here is a todo comment.